.husky/pre-commit

```diff
@@ -1,5 +1 @@
-#!/usr/bin/env sh
-. "$(dirname -- "$0")/_/husky.sh"
-
 yarn fmt
-git add -A
```

Comment:
This would include all Prettier-formatte files when you commit (good) but also include files you didn't add to that commit (bad). I'll change it in a new PR to use `lint-staged` or something similar.

Reply:
Looking forward to this change
I'm not a big fan of the `git add -A` on prettier pre-commits 🤣